console/config/mock/section.py

```diff
@@ -17,6 +17,11 @@
 # under the License.
 #
 
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+from __future__ import absolute_import
+
 import json
 import re
 from schema import Schema
```

Comment:
I think this needs to be:
from .schema import Schema
since the schema.py is in the same directory as the section.py file.

Reply:
Good catch - I'll push a fix...